apps/aehttp/src/aehttp_app.erl

```diff
@@ -99,8 +99,8 @@ start_http_api(Target, LogicHandler) ->
             {ip, ListenAddress},
             {num_acceptors, PoolSize}],
     Env = #{env => #{dispatch => Dispatch},
-            middlewares => [cowboy_router,
-                            aehttp_cors_middleware,
+            middlewares => [aehttp_cors_middleware,
```

Comment:
Why was the order a problem?

Reply:
Because by fixing the ordering we are fixing the problems with headers, our `cors` headers will **always** be added, instead of default headers of the cowboy router.